### ISSUE TITLE
cannot set attribute stroke and fill

### ISSUE THREAD

**Issue by ingaYu** [konvajs/react-konva#267]
It like a bug, I draw a group on my page and it shows some simple sharp
```
<Group key={index} id={`rect_group_${index}`} x={left} y={top} onClick={(e) => {
    e.target.stroke('#00334b')
    e.target.setAttr('fill', '#00334b')
}}>
     <Rect width={rectWidth} height={rectHeight} stroke='#e6e4dd' strokeWidth={6}/>
     <Circle width={26} height={26}  fill='#e6e4dd' shadowColor='black' shadowBlur={2} 
           shadowOpacity={0.17}/>
</Group>
```
it works well, but can't handle click event right.

The phenomenon is as follows：

- click Reck:  The color of the fill has changed but the color of the stroke has not changed. Is it because I have already set Rect stroke up?
- click Circle:  Contrary to the above， Is it because I have already set Circle fill up?

**Comment by lavrton** [comment]
It is better not to edit attributes manually. Currently `react-konva` has full control over attributes. If you changed `fill` manually, `react-konva` may reset it back to the value provided in `render` function.
If you want to change attributes manually, then don't define them in `render`.

**Comment by ingaYu** [comment]
yes, I resolve it.  Just set `stroke` and `fill` values in the state.  thank you!
